### ISSUE TITLE
Veronte / Veronte / Rawdat : RTK working

### ISSUE THREAD

**Issue by gir2** [gir2/prueba_issues#14]
**Reported by admin on 5 Mar 2013 15:46 UTC**
None

**Comment by gir2** [comment]
**Modified by dbs on 28 Aug 2013 12:20 UTC**

**Comment by gir2** [comment]
**Comment by dbs on 28 Aug 2013 12:21 UTC**
Add position of ground: set_wrt(const Rvector3& rgh0)

**Comment by gir2** [comment]
**Comment by jbm <jbm@embention.com> on 25 Sep 2013 15:13 UTC**
In [3dcfe4ffd641cad0eb7577270c8ed6a3eb0dc30f/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="3dcfe4ffd641cad0eb7577270c8ed6a3eb0dc30f"
Refs #13, closes #15.
 - Added extra configuration to ECAP to allow it to work as GPS time pulse.

Refs #236:
 - Time precission internally working at cpu clock precission (150MHz clock).
 - Clock and Chrono adapted.

Refs #227:
 - Rawdat now uses 64 bits Real for both carrier and pseudoranges (removed d2fp, and subfp) instead of Uint64.
 - Serialization of RAW from GND changed (values out of str range found on previous version).

Changed JFX project properties.
```

**Comment by gir2** [comment]
**Modified by dbs on 13 Sep 2014 09:58 UTC**
